daiquiri/metadata/assets/js/components/Schemas.js

```diff
@@ -44,40 +47,56 @@ const Schemas = ({ schemas, activeItem, setActiveItem, getTooltip, onDoubleClick
         }
 
         setOpenTable(table)
+        if (refListTables.current) {
```

Comment:
This scrolls the `dq-browser-list` for tables/columns to its top when it is set to a different schema/table, right? (I have to few items in my test setup.)

Reply:
Yes, this scrolls the list of the tables to the top if a schema is clicked. The column list works similar.